packages/payment/src/models/payment-session.ts

```diff
@@ -22,13 +28,15 @@ export default class PaymentSession {
   id: string
 
   @Property({ columnType: "text" })
-  currency_code: string
+  currency_code: string | number
```

Comment:
question: Why have we added `number` here?

Reply:
My mistake, completely involuntary